### ISSUE TITLE
Camera connector on Lichee Pi Zero

### ISSUE THREAD

**Issue by cameronelliott** [Unturned3/h264enc_demo#1]
removed

**Comment by cameronelliott** [comment]
Do you have any suggestions on trying to get this working on a Lichee Pi Zero?
It seems like I need to get a Pi Zero with the headers installed and figure out some wiring to an OV5640, right ? As far as I can tell there is no 15 pin Mipi Csi on the Lichee Pi Zero, but I just started researching this.

**Comment by cameronelliott** [comment]
Okay, I have been digging around, it looks like on the larger Lichee Pi Zero, there is a dual row 24 pin connector, which it looks like you can buy the OV5640 with!

**Comment by Unturned3** [comment]
Yes, you need the Lichee Pi Zero Dock (i.e. the "larger" one). Apparently you can also connect a camera to the small board via the LCD bus using an adapter, but I have not tried it.
Also, I did not use the dual-row 24 pin connector. That is for the MIPI-CSI2 bus. I used DVP (the single-row 24 pin connector).

**Comment by cameronelliott** [comment]
Oh wow! This was so helpful! Yeah, I see it now, J5 is a 24 pin DVP connector. For some reason I thought the OV5640 was MIPI only, but I just checked the datasheet, and it appears to support both MIPI and DVP. It looks like I may be able to get the OV5640 for around $2 on Taobao (in qty), if I can ever figure out how to register properly. lol.

Thank you so much! Without your help I probably would have ordered a dock unit, but with one or two OV5640 MIPI units! Ouch! Then I would have to wait or pay for the DVP 5640 units if/when I figured out why stuff wasn't working. :smile:
